README.md

```diff
@@ -27,6 +27,8 @@ sudo wget https://github.com/sqshq/sampler/releases/download/v1.0.3/sampler-1.0.
 sudo chmod +x /usr/local/bin/sampler
 ```
 Note: `libasound2-dev` system library is required to be installed for Sampler to play a [trigger](https://github.com/sqshq/sampler#triggers) sound tone. Usually the library is in place, but if not - you can install it with your favorite package manager, e.g `apt install libasound2-dev`
+#### Packaging status
+- [Fedora](https://apps.fedoraproject.org/packages/sampler) `sudo dnf install sampler` (F31+)
```

Comment:
Should not be command: `dnf install golang-github-sqshq-sampler`?

Reply:
Indeed. Thanks a lot. My first Golang package. 🤷🏻‍♂️